validation.js

```diff
@@ -0,0 +1,39 @@
+// let errorSpan = document.getElementById('error');
+// let email = document.getElementById('email');
+// let form = document.getElementsByTagName('form');
+
+
+// function validateEmail(){
+//     const lowerCase = /^[a-z0-9.@]+$/;
+//     let result = lowerCase.test(email.value);
+//     return result;
+// }
+
+// function showError(){
+//     errorSpan.style.display = 'block';
+//     errorSpan.textContent = 'Email address must be lowercase'; 
```

Comment:
- [ ] All the code in this file is commented out, there fore there's no functionality to sustain the validation. Please kindly work on the required validation

Reply:
thank you 😶 
this is because the last minute checks something. and forget to make the code uncomment.